kv/client.go

```diff
@@ -52,8 +52,8 @@ type StoreConfig struct {
 // Config is config for a KVStore currently used by ring and HA tracker,
 // where store can be consul or inmemory.
 type Config struct {
-	Store       string `yaml:"store"`
-	Prefix      string `yaml:"prefix"`
+	Store       string `yaml:"store" category:"advanced"`
```

Comment:
I think changing KV backend store should be considered basic. Based on this assumption, I've the rest of the comments.

Reply:
@pracucci done.